packages/emotion/src/index.js

```diff
@@ -96,7 +96,12 @@ const processStyleValue = (key, value) => {
   if (value === undefined || value === null || typeof value === 'boolean')
     return ''
 
-  if (unitless[key] !== 1 && !isNaN(value) && value !== 0) {
+  if (
+    unitless[key] !== 1 &&
+    key.indexOf('--') !== 0 &&
```

Comment:
Could you change this to `key.charCodeAt(1) !== 45`(45 is the char code for -) since this is a very hot path and `charCodeAt` is much faster.

Reply:
Do you mean checking both first two chat codes?

Reply:
No, there's no need to since there are no regular css properties with a dash as the second character.